src/troute-network/troute/HYFeaturesNetwork.py

```diff
@@ -247,6 +253,9 @@ def __init__(self,
                     value_dict, 
                     bmi_parameters,
                     )
+            #FIXME: See FIXME above.
+            if not from_files_copy:
+                from_files=False
 
             # Preprocess network objects
             self.preprocess_network(flowpaths)
```

Comment:
At L269, after changing use_preprocessed_data to False, I got an error saying that 
PermissionError: [Errno 13] Permission denied: '/home/APD/inland_hydraulics/Apr2023_testcase/preprocess/Extended_AnA_output.npy'

Reply:
That's a permissions issue on apd-dev, not necessarily a t-route issue. But curious why that came up. Do you have `preprocess_only:` set to `True`?

Reply:
I put False to both preprocess_only and use_preprocessed_data.